layouts/index.html

```diff
@@ -235,9 +232,9 @@ <h5>Developing world</h5>
             </div>
             <h5>Blockchains</h5>
             <p>
-              IPFS lets you address large amounts of data and place the immutable,
-              permanent links into blockchain transactions. This timestamps
-              and secures content without having to put the data itself on the chain.
+              With IPFS, you can address data and put immutable,
+              permanent links in transactions &mdash; timestamping
+              and securing content without having to put the data itself on-chain.
```

Comment:
Don't want to lose `large amounts of`, looks good otherwise.

Reply:
Done.